CMake/External_OpenCV.cmake

```diff
@@ -21,6 +24,23 @@ else()
   set(fletch_ENABLE_OpenCV_FFmpeg CACHE INTERNAL FALSE)
 endif()
 
+# Allow OpenCV's VTK option to be explicitly turned off
+if(fletch_ENABLE_VTK)
+  # By default turn opencv-VTK support off, even when building with VTK
+  option(fletch_ENABLE_OpenCV_VTK "Build OpenCV with VTK support" FALSE )
+  mark_as_advanced(fletch_ENABLE_OpenCV_VTK)
+else()
+  set(fletch_ENABLE_OpenCV_VTK CACHE INTERNAL FALSE)
```

Comment:
@Erotemic , It does exist for others, yes. The FFmpeg one always bothered me and I think the CUDA one was needed because it would break the build under certain uncontrollable conditions, like CUDA version. I don't know if I care if it remains for a while because of build related issues, but it we are able to, as a group, decide if it should be on/off, then that is better.